src/utils.js

```diff
@@ -665,10 +666,16 @@ function getCompileFn(implementation, options) {
 
   if (isNewSass) {
     if (options.api === "modern") {
-      return (sassOptions) => {
+      return async (sassOptions) => {
         const { data, ...rest } = sassOptions;
 
-        return implementation.compileStringAsync(data, rest);
+        if (!sassEmbeddedCompiler) {
+          // Create a long-running compiler process that can be reused
+          // for compiling individual files.
+          sassEmbeddedCompiler = await implementation.initAsyncCompiler();
```

Comment:
I'm not sure about the lifetime of a webpack loader. If webpack provides some hook for this, we could close the compiler process by calling `sassEmbeddedCompiler.dispose()`.

Reply:
Also will be great to make some benches, for example for bootstrap to undestand how it is better

Reply:
It is slow for perf, initial start will be very long and we don't need subprocess

Reply:
I've implemented your suggestions. Re:

> Also will be great to make some benches, for example for bootstrap to undestand how it is better

is there a pre-existing repo or setup I can use to benchmark this?

Reply:
No, let's just check it, I will put it in release notes

Reply:
For what it's worth, I don't have the exact numbers but on a larger project I'm working on, `modern-compiler` is up to 10 seconds faster than `modern`. Also `modern` would just crash when compiling a changed file that's included in various other files, whereas with `modern-compiler` it works fine.